### ISSUE TITLE
Multisite - NoReverseMatch

### ISSUE THREAD

**Issue by leesolway** [nephila/djangocms-blog#281]
Scenario -- 

**Site 1** 
- Create an App Conf / Hook App to page
- Site 1 working as expected 

**Site 2**
- Forced to create a new App Conf and assign to page, as an AppHook Conf is unique. 
- Site 2 returning error: NoReverseMatch: u'djangocms_blog' is not a registered namespace

I assume this is because of having to define the namespace on the new App conf.. 

```
Internal Server Error: /about-cmac/news/
Traceback (most recent call last):
  File "/var/www/example/env/lib/python2.7/site-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/response.py", line 135, in rendered_content
    content = template.render(context, self._request)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 210, in render
    return self._render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/test/utils.py", line 96, in instrumented_test_render
    return self.nodelist.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/test/utils.py", line 96, in instrumented_test_render
    return self.nodelist.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/test/utils.py", line 96, in instrumented_test_render
    return self.nodelist.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/test/utils.py", line 96, in instrumented_test_render
    return self.nodelist.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/test/utils.py", line 96, in instrumented_test_render
    return self.nodelist.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/classytags/core.py", line 146, in render
    return self.render_tag(context, **kwargs)
  File "/var/www/example/env/lib/python2.7/site-packages/sekizai/templatetags/sekizai_tags.py", line 83, in render_tag
    rendered_contents = nodelist.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/classytags/core.py", line 146, in render
    return self.render_tag(context, **kwargs)
  File "/var/www/example/env/lib/python2.7/site-packages/cms/templatetags/cms_tags.py", line 631, in render_tag
    rendered_contents = nodelist.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/classytags/core.py", line 146, in render
    return self.render_tag(context, **kwargs)
  File "/var/www/example/env/lib/python2.7/site-packages/cms/templatetags/cms_tags.py", line 1100, in render_tag
    content = render_placeholder(placeholder, context, name_fallback=code, default=nodelist)
  File "/var/www/example/env/lib/python2.7/site-packages/cms/plugin_rendering.py", line 188, in render_placeholder
    content.extend(render_plugins(plugins, context, placeholder, processors))
  File "/var/www/example/env/lib/python2.7/site-packages/cms/plugin_rendering.py", line 101, in render_plugins
    out.append(plugin.render_plugin(context, placeholder, processors=processors))
  File "/var/www/example/env/lib/python2.7/site-packages/cms/models/pluginmodel.py", line 193, in render_plugin
    return render_plugin(context, instance, placeholder, template, processors, current_app)
  File "/var/www/example/env/lib/python2.7/site-packages/cms/plugin_rendering.py", line 74, in render_plugin
    content = template.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 210, in render
    return self._render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/test/utils.py", line 96, in instrumented_test_render
    return self.nodelist.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/defaulttags.py", line 224, in render
    nodelist.append(node.render(context))
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/defaulttags.py", line 507, in render
    six.reraise(*exc_info)
  File "/var/www/example/env/lib/python2.7/site-packages/django/template/defaulttags.py", line 493, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/var/www/example/env/lib/python2.7/site-packages/django/core/urlresolvers.py", line 574, in reverse
    key)
NoReverseMatch: u'djangocms_blog' is not a registered namespace
```

**Comment by yakky** [comment]
@leesolway did you restarted after adding the second namespace?

Also, probably the uniqueness of apphook config across a mutisite project is wrong (see https://github.com/divio/django-cms/pull/5458)

**Comment by leesolway** [comment]
@yakky Awesome, that should do it. 

I though the app should restart the wsgi process now. I will double check this and confirm the server restarted.

**Comment by yakky** [comment]
@leesolway fixed by divio/django-cms#5458, which will be released in 3.3.1
